### PR TITLE
Progress Bar Design Documentation

### PR DESCRIPTION
**Pull request by rachael-phillips**
## Description
* This is the design documentation for progress bar. The pull request is to review the progress bar design documentation.

## Changes
* All design documentation for progress bar including supporting images

## Contribution Submission Checklist
- [ ] A GitHub issue was created and reviewed with the PatternFly community.
- [ ] The pattern design overview.md and design.md templates were used to provide all information, relevant to this design.
- [ ] Any new files have been added to the folder for that pattern.
- [ ] The folder, for brand new patterns, has been titled with the name of the pattern.
- [ ] All images are in an "img" folder and embedded images have the correct filepath.
- [ ] The overview.md, design.md and site.md files, for this design have been commited and pushed to the patternfly-design repository, for review.

### CODE REVIEW

File: patternfly-4/components/progress-bar/design.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+### When not to use
+Do not use an indeterminate progress bar to indicate measurable progress. Users need progress bars to represent measurable progress if at all possible. Use an indeterminate progress bar only if progress is not measurable.
+
+Do not use a progress bar when a spinner should be used instead. A spinner should not be used for long processes. Only use a spinner if a process will take 1-4 seconds. A spinner will never replace a determinate progress bar as the spinner does not provide helpful information about loading time to a user.
+
+Do not use a spinner and a progress bar together. Use one progress measure at a time.
+
+## How to use progress bar in context
+![progress-bar](img/progress-bar-do-dont.png.png)
```
Comment (mcarrano) [inline patternfly-4/components/progress-bar/design.md]: This image is not displaying for some reason.

**Comment by rachael-phillips** [comment]
Hi Matt,

Thank you for pointing that out! No, I didn't know about Preview mode in
GitHub, so I will definitely take a look at that!

Rachael

**Comment by rachael-phillips** [comment]
Thank you for reviewing this so promptly! You rock!

**Comment by rachael-phillips** [comment]
Woohoo! Thank you Matt for all of your help along the way!

**Pull request merged**